modules/jetstream-maxtext-deployment/main.tf

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  deployment_template         = "${path.module}/templates/deployment.yaml.tftpl"
+  service_template            = "${path.module}/templates/service.yaml.tftpl"
+  podmonitoring_template      = "${path.module}/templates/podmonitoring.yaml.tftpl"
+  cmsa_jetstream_hpa_template = "${path.module}/templates/custom-metrics-stackdriver-adapter/hpa.jetstream.yaml.tftpl"
+}
+
+resource "kubernetes_manifest" "jetstream-deployment" {
+  count = 1
+  manifest = yamldecode(templatefile(local.deployment_template, {
+    maxengine_server_image      = var.maxengine_server_image
+    jetstream_http_server_image = var.jetstream_http_server_image
+    load_parameters_path_arg    = format("load_parameters_path=gs://%s/final/unscanned/gemma_7b-it/0/checkpoints/0/items", var.bucket_name)
```

Comment:
Is this "gs://%s/final/unscanned/gemma_7b-it/0/checkpoints/0/items' the guaranteed format for the parameters path?